cellprofiler/modules/namesandtypes.py

```diff
@@ -314,7 +314,7 @@ def create_settings(self):
                 doc=LOAD_AS_CHOICE_HELP_TEXT)
 
         self.volumetric = cps.Binary(
-            text="Volumetric",
+            text="Data is 3D",
```

Comment:
IIUC, one might have 3d (xyz) data but want to process it slice by slice, in such a case the answer here would be "no". Now, we can just assume that in this case the person will know they are over-riding 'normal' 3D processing behavior and so maybe they will feel ok checking 'no'.